swdb_2019_tools/sanity_check.py

```diff
@@ -0,0 +1,56 @@
+def sanity_check(mat,xname='x-axis(unknown units)',yname = 'y-axis(unknown units)', zname = 'z-axis(unknown units)',title = 'Unknown matrix',ncols=5):
```

Comment:
i'd reccomend a different name, and letting people pass in the axis they want to plot on.